sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/AnalyzeSentimentAction.java

```diff
@@ -0,0 +1,136 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.textanalytics.models;
+
+import com.azure.core.annotation.Fluent;
+
+/**
+ * The option bags to use when analyzing sentiment as an action.
```

Comment:
Consider changing all docstrings for the Action types to something like:
`Configurations that allow callers to specify details about how to execute an Analyze Sentiment action in a set of documents.`